src/m_files.cc

```diff
@@ -942,6 +943,15 @@ bool Instance::M_ParseEurekaLump(const Wad_file *wad, bool keep_cmd_line_args)
 				gLog.printf("  trying: %s\n", res.c_str());
 			}
 
+                        if (! FileExists(res))
+                        {
+                                //now try relative to PWAD by prepending PWAD path
+                                SString pwadPath = FilenameGetPath(wad->PathName());
+                                pwadPath += DIR_SEP_STR;
+                                res = (pwadPath += resBackup);
+                                gLog.printf("  trying: %s\n", res.c_str());
+                        }
+
```

Comment:
Are you adding the rule that entire paths should also be attempted next to the PWAD's path, not just mere basenames (as it is in the `if` block above)?

Reply:
Yes. In the most ideal situation of the resources being in a folder in the same directory as the project pwad, I felt it would be acceptable to just append the whole path to the pwad's path. FLTK's relative path function returns a path relative to the current working directory, so when we chdir to the pwad directory, it'll return a relative directory something like "resources/foo.wad".